R/mod_create_table.R

```diff
@@ -0,0 +1,542 @@
+#' create_table UI Function
+#'
+#' @description A shiny Module.
+#'
+#' @param id,input,output,session Internal parameters for {shiny}.
+#'
+#' @noRd
+#'
+#' @importFrom DT DTOutput
+#' @importFrom DT renderDT
+#' @importFrom DT datatable
+#' @importFrom RSQLite dbExecute
+
+mod_create_table_ui <- function(id) {
+  ns <- NS(id)
+  tabPanel(
+    title = "Create Table",
+    br(),
+    fluidRow(p(h2(
+      strong("Create a new Table")
+    ))),
+    fluidRow(
+      textInput(
+        inputId = ns("new_table_name"),
+        label = h4(strong("Enter Table Name"))
+      ),
+      DT::DTOutput(ns("display_new_table")),
+      column(width = 2,
+             actionButton(
+               inputId =  ns("add_column"),
+               label = "Add a New Column"
+             )),
+      column(
+        width = 3,
+        actionButton(inputId = ns("remove_columns"),
+                     label = "Remove Selected Columns")
+      ),
+      column(width = 2,
+             actionButton(
+               inputId = ns("reset_columns"),
+               label = "Reset Columns"
+             )),
+      column(width = 2,
+             actionButton(
+               inputId = ns("create_new_table"),
+               label = "Create New Table"
+             ))
+    ),
+    br(),
+    HTML(
+      '<hr style="height:2px;
```

Comment:
use hr() function and seperate CSS to css file. Searching in R script for CSS change is inefficient.